package.json

```diff
@@ -59,11 +59,12 @@
     "istanbul": "^0.4.5",
     "lodash": "^4.17.4",
     "moment": "^2.19.2",
-    "mongoose": "^4.7.6",
+    "mongoose": "^4.13.14",
     "mongoose-shortid-nodeps": "git+https://github.com/leeroybrun/mongoose-shortid-nodeps.git",
     "moodle-client": "^0.4.0",
     "node-sass": "^4.2.0",
     "nodemailer": "^3.0.2",
+    "npm": "^6.1.0",
```

Comment:
Es macht glaube wenig Sinn, das npm-Modul in die package.json zu schreiben. Denn ohne npm funktioniert die package.json überhaupt nicht. Man muss sowieso npm installieren, bevor die package.json initiiert wird. Das kann also raus